src/decision/include/DecisionNode.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: March 17, 2018
+ * Description:
```

Comment:
We should make sure `Description` is filled in on all files.

Reply:
done